### ISSUE TITLE
Need example

### ISSUE THREAD

**Issue by xgrommx** [vslinko/webpack-amok-dev-server#1]
Hello @vslinko. I need example how can I use it =)

**Comment by vslinko** [comment]
OK, I'll try to write some docs today.

**Comment by vslinko** [comment]
I'm done with README. Is it enough to start?

**Comment by xgrommx** [comment]
About `cwd: path.join(__dirname, '.app', 'public')` here should be my entry point like a `index.js`?

**Comment by vslinko** [comment]
No, here should be a path to compile directory. Same as `output.path` in your webpack configuration.

**Comment by xgrommx** [comment]
@vslinko Maybe will be better create a simple app with `index.html` which include `index.js`

**Comment by vslinko** [comment]
It's good idea with simple example but I don't have enough time on this week.
You could help me or just wait some time.

**Comment by xgrommx** [comment]
@vslinko Why not =) But i need explain which an url in index.html I need include. For example in react-hot-loader I use :

``` js
    entry: [
        'webpack-dev-server/client?http://localhost:3000',
        'webpack/hot/only-dev-server',
        './src/index'
    ],

    output: {
        path: path.join(__dirname, 'build'),
        filename: 'bundle.js',
        publicPath: '/static/'
    },
```

And in development mode I include my script in index.html so:

``` html
<script src="/static/bundle.js"></script>
```

If you to explain me how can I include a script I will try create a simple example.

**Comment by vslinko** [comment]
> But i need explain which an url in index.html I need include

You should use `/static/bundle.js` as usual.
Also you should remove `webpack-dev-server` and `webpack/hot` from your `entry` because Amok uses own system.

This is config from my private project. You can use it as example.

``` js
var webpack = require('webpack')
var path = require('path')

module.exports = {
  entry: path.join(__dirname, 'src', 'frontend'),

  output: {
    path: path.join(__dirname, '.app', 'public'),
    filename: 'frontend.js',
    publicPath: '/',
    library: 'frontend',
  },

  module: {
    loaders: [/* cutted */]
  },

  amokDevServer: {
    cwd: path.join(__dirname, '.app', 'public'),
    publicPath: '/',
    filename: 'frontend.js',
    proxy: {
      '*': 'http://localhost:3000',
    },
  },
}
```

**Comment by vslinko** [comment]
Also WebpackAmokDevServer doesn't serve static files itself. I would like to add this feature before example because I don't wont to use proxy in example.

**Comment by vslinko** [comment]
So @xgrommx If you have little more time then you could configure `express.static` middleware.

**Comment by vslinko** [comment]
See #2
